modules/social_features/social_comment/social_comment.install

```diff
@@ -13,67 +13,47 @@ use Drupal\user\Entity\Role;
  * Perform actions related to the installation of social_comment.
  */
 function social_comment_install() {
-  // Set some default permissions.
-  _social_comment_set_permissions();
+  // Grant the default permissions for this feature.
+  user_role_grant_permissions(
+    'verified',
+    [
+      'access comments',
```

Comment:
`access comments` - allows view comments.

I did not add it for Authenticated users (unverified) because previously Anonymous users had no this permission by default and for now, Authenticated users have mostly the same permissions as Anonymous users previously. 

So, if you think that we need to add this permission to unverified users we can do that.